pkg/database/user.go

```diff
@@ -223,6 +223,29 @@ func (u *User) Stats(db *Database, realmID uint, start, stop time.Time) ([]*User
 	return stats, nil
 }
 
+// DeleteUser deletes the user entry.
+func (db *Database) DeleteUser(u *User) error {
+	return db.db.Delete(u).Error
+}
+
+// ListUsers returns a list of all users sorted by name.
+func (db *Database) ListUsers() ([]*User, error) {
+	var users []*User
+	if err := db.db.
+		Model(&User{}).
+		Where("admin IS FALSE").
```

Comment:
Why exclude admins?

Reply:
There are two pages: Admin users and Users. The delete action deletes the user here, but the admin users delete button just de-privileges the user by removing the admin bit.
I'm considering merging things or moving actions to a sub-page together, but for now I'm trying to keep them separate